src/GitHub.InlineReviews/Services/InlineCommentPeekService.cs

```diff
@@ -51,18 +51,18 @@ class InlineCommentPeekService : IInlineCommentPeekService
             {
                 if (diffModel.ViewType == DifferenceViewType.InlineView)
                 {
-                    // If we're displaying a diff in inline mode, then we're in the left buffer if
-                    // the point can be mapped down to the left buffer.
+                    // If we're displaying a diff in inline mode, then we need to map the point down
```

Comment:
// If we're displaying a diff in inline mode, then we need to map the point down